src/components/bookmarks/BookmarksPanel.tsx

```diff
@@ -200,6 +199,9 @@ export default observer(BookmarksPanel);
 
 interface BookmarkItemProps {
 	bookmark: BookmarkedItem;
+	messages?: EventMessage[];
```

Comment:
i cant see these props being used anywhere, and passing list of messages to a bookmark doesnt seem like a good idea

Reply:
that was a logic about highlighting elements if they're exist in events or messages panel. Removed from bookmark components and implemented in actual components: https://github.com/th2-net/th2-rpt-viewer/pull/322/commits/61548c3c2c7f81cf33a68c8a83f8b860fbffee13